lib/lorekeeper/fast_logger.rb

```diff
@@ -21,8 +21,29 @@ def initialize(file)
       @file = file # We only keep this so we can inspect where we are sending the logs
     end
 
-    LOGGING_METHODS = [:debug, :info, :warn, :error, :fatal]
-    METHOD_SEVERITY_MAP = { debug: DEBUG, info: INFO, warn: WARN, error: ERROR, fatal: FATAL }
+    LOGGING_METHODS = [
+      :debug,
+      :info,
+      :warn,
+      :error,
+      :fatal
+    ].freeze
+
+    METHOD_SEVERITY_MAP = {
+      debug: DEBUG,
+      info: INFO,
+      warn: WARN,
+      error: ERROR,
+      fatal: FATAL
+    }.freeze
+
+    SEVERITY_NAMES_MAP = {
+      DEBUG => 'debug',
+      INFO => 'info',
+      WARN => 'warn',
```

Comment:
this must be warning 😐

Reply:
😱 
will fix.

Reply:
https://github.com/JordiPolo/lorekeeper/pull/9/commits/9f0e56a9d602facfa1b772560ce91356b2e9251a